external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/KafkaSpoutConfig.java

```diff
@@ -230,8 +237,37 @@ public Builder(Map<String, Object> kafkaProps, KafkaSpoutStreams kafkaSpoutStrea
         }
 
         public KafkaSpoutConfig<K,V> build() {
+            validate();
             return new KafkaSpoutConfig<>(this);
         }
+
+        /**
+         * Defines whether the consumer manages partition manually.
+         * If set to true, the consumer behaves like a simple consumer, otherwise it will rely on kafka to do partition assignment.
```

Comment:
Consider changing this to "If set to true, the spout instances are assigned partitions round-robin", since readers maybe don't know what it means to behave like a simple consumer.